### ISSUE TITLE
Incorrect ONNX exported TRT engine results

### ISSUE THREAD

**Issue by nrakltx** [NVIDIA/TensorRT#2798]
## Description

I am attempting to optimize model inference using TensorRT.
I have an ONNX model that I am exporting to TensorRT (using [this](https://github.com/NVIDIA/TensorRT/issues/2751) code).
The exporting is successful but the output is incorrect - ORT and TRT return different results!
What could be causing this? I am exporting using FP16 (the ONNX is already FP16), could inaccuracies and clamping be doing enough damage to cause this behaviour?

## Environment

**TensorRT Version**: 8.6.0
**NVIDIA GPU**: Telsa T4
**NVIDIA Driver Version**: 530.30.02
**CUDA Version**: 12.1
**CUDNN Version**: 
**Operating System**: Debian 11
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 3.9
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

The ONNX and engine files are in [this](https://drive.google.com/drive/folders/1U4VcrHoRnO6cHcOzccn4plZm0YB65AKO?usp=share_link) Drive folder.

## Steps To Reproduce
Run the following code in Python 3.9 with a GPU machine.
```Python
import torch
import onnxruntime as ort
from polygraphy.backend.trt import TrtRunner, engine_from_bytes

ort_session = ort.InferenceSession(
    "model_800_1344_fp16.onnx", providers=["CUDAExecutionProvider", "CPUExecutionProvider"]
)

def onnx_model(*args, **kwargs):
    if len(args) != 0:
        inputs = {f"INPUT{i}": input for i, input in enumerate(args)}
    else:
        inputs = kwargs
    inputs = {k: v.detach().cpu().numpy() for k, v in inputs.items()}

    np_outputs = ort_session.run(None, input_feed=inputs)

    torch_outputs = [torch.from_numpy(output).cuda() for output in np_outputs]

    if len(torch_outputs) == 1:
        return torch_outputs[0]
    return torch_outputs

inputs = torch.randn((1, 3, 800, 1344), dtype=float_type, device=device)

with open("model_800_1344_fp16_2.engine", "rb") as f:
    engine = engine_from_bytes(f.read())
with TrtRunner(engine, "test", 0) as runner:
    trt_model = lambda x: runner.infer({"INPUT0": x})

    b = onnx_model(inputs)
    c = trt_model(inputs.cpu().numpy())
torch.testing.assert_close(b[0].detach().cpu().numpy(), c["OUTPUT0"])
```

**Comment by nrakltx** [comment]
Yes sorry I copied an older version of the code snippet, I edited the original issue and it should now work. Thanks for the fast response!

**Comment by zerollzeng** [comment]
Thanks, I've filed internal bug 4052062 for this. Thanks!

One more question, the above polygraphy log is generated using 8.6 EA?

**Comment by nrakltx** [comment]
Thank you! This is 8.6 EA indeed.

**Comment by ttyio** [comment]
This is fixed in 8.6GA, closing and thanks all!
